contrib/jenkins_tests/copyrights.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -xeEl
+
+source $(dirname $0)/globals.sh
+
+if [ -z "$WORKSPACE" ]; then
+    echo "ERROR: WORKSPACE variable is empty"
+    exit 1
+fi
+
+if [ ! -d "$WORKSPACE" ]; then
+    echo "ERROR: $WORKSPACE does not exist"
+    exit 1
+fi
+
+HEADER_CHECK_TOOL=/opt/nvidia/header_check.py
+if [ ! -f "${HEADER_CHECK_TOOL}" ]; then
+    echo "ERROR: ${HEADER_CHECK_TOOL} doesn't exist!"
+    exit 1
+fi
+
+cpp_files='        "extensions": [".c", ".cc", ".cpp", "c++", ".h", ".hpp", ".cs", ".inl", ".l", ".y"],'
+sed -i "s/.*\"extensions\": \[\"\.c\".*/$cpp_files/g" /opt/nvidia/ProjectConfig/header-types.json
+
+cat /opt/nvidia/ProjectConfig/header-types.json
+
+${HEADER_CHECK_TOOL} \
+  --config ${WORKSPACE}/contrib/jenkins_tests/copyright-check-map.yaml \
+  --path ${WORKSPACE} \
+  --git-repo ${WORKSPACE} | tee copyrights.log
+exit_code=$?
+echo "exit_code=${exit_code}"
+# Correct error code is not returned by the script
+set +eE
+grep -rn ERROR copyrights.log
+exit_code=$?
+set -eE
```

Comment:
If tool is failed to start then error log can be empty. So I added check for first exit code too. BTW, have you tried `set -o pipefail` to check exit codes for all commands in pipeline?
```suggestion
exit_code=$?
echo "exit_code=${exit_code}"
if [ $exit_code -eq 0 ]
then
    # Script does not return error flag in exit code
    set +eE
    grep ERROR copyrights.log
    exit_code=$?
    set -eE
fi
```